qtfred/src/ui/widgets/sexp_tree.cpp

```diff
@@ -2749,8 +2749,20 @@ void sexp_tree::update_help(QTreeWidgetItem* h) {
 		}
 	}
 
+	// Node comments are not yet implemented in qtFRED, so just adding some base code here
+	// that can be used when the feature is completed - Mjn
+
+	//int thisIndex = event_annotation_lookup(h);
+	SCP_string nodeComment;
+
+	//if (thisIndex >= 0) {
```

Comment:
Comments from FRED's sexp_tree.cpp should also apply to qtFRED's sexp_tree.cpp where applicable.

Reply:
done